benchmarks/AuraDiBench.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace PhpBench\Benchmarks\Container;
+
+use Aura\Di\ContainerBuilder;
+
+/**
+ * @Groups({"aura-di"}, extend=true)
+ */
+class AuraDiBench extends ContainerBenchCase
+{
+    private $container;
+
+    public function benchGetUnoptimized()
```

Comment:
Also note that there is currently a [bug](https://github.com/phpbench/phpbench/pull/385) which I discovered today which makes `@Skip` misbehave incase you have any issues.